src/interfaces/Config.ts

```diff
@@ -83,6 +91,12 @@ export interface IChartConfig extends IMeasuresStackConfig {
     secondaryChartType?: VisualizationObject.VisualizationType;
     forceDisableDrillOnAxes?: boolean;
     disableDrillUnderline?: boolean;
+
+    // config for GeoChart
+    center?: [number, number];
+    geoDataIndex?: IGeoDataIndex;
+    tooltipText?: VisualizationInput.IAttribute;
+    zoom?: number;
```

Comment:
How about putting four new confg items to one interface and `IChartConfig extends IMeasuresStackConfig, IGeoChartConfig`